create tech note about CRediTas new package

jospueyo opened the pull request:
I added author file and the files for the tech note.

jospueyo commented:
@ropensci/blog-editors I'm not sure if I skipped some step of the process.

yabellini commented:
Hi @jospueyo! Thanks for writing this technote about your package. The next step is for us to review it and reply with our feedback.
Here is a checklist with the steps for authors, please check the one you complete):

* [ ] I have read the Content Guidelines.
* [ ] I have read the Technical Guidelines.
* [ ] I used or followed the R Markdown or Markdown template.
* [ ] I have followed the Style Guide.
* [ ] I created or updated my author metadata with correct folder name.
* [ ] I have added relevant tags after browsing existing tags (including "community" tag).
* [ ] I have added the "tech notes" tag if this is a technote.
* [ ] I ran `roblog::ro_lint_md()` on index.md (optional).
* [ ] I ran `roblog::ro_check_urls()` on index.md (optional).
* [ ] I ran a spell-check on index.md.
* [ ] I have added the tags - Software Peer Review, my-packagename.
* [ ] I have added the package-version YAML tag.
* [ ] I have added acknowledgement of the reviewers' work (with links to reviewers).
* [ ] I have added a link to the software peer review thread.

jospueyo commented:
> Hi @jospueyo! Thanks for writing this technote about your package. The next step is for us to review it and reply with our feedback. Here is a checklist with the steps for authors, please check the one you complete):
> 
> * [x]  I have read the Content Guidelines.
> * [x]  I have read the Technical Guidelines.
> * [x]  I used or followed the R Markdown or Markdown template.
> * [x]  I have followed the Style Guide.
> * [x]  I created or updated my author metadata with correct folder name.
> * [x]  I have added relevant tags after browsing existing tags (including "community" tag).
> * [x]  I have added the "tech notes" tag if this is a technote.
> * [ ]  I ran `roblog::ro_lint_md()` on index.md (optional).
> * [ ]  I ran `roblog::ro_check_urls()` on index.md (optional).
> * [x]  I ran a spell-check on index.md.
> * [ ]  I have added the tags - Software Peer Review, my-packagename.
> * [ ]  I have added the package-version YAML tag.
> * [ ]  I have added acknowledgement of the reviewers' work (with links to reviewers).
> * [x]  I have added a link to the software peer review thread.

steffilazerte commented:
Hi @jospueyo 

> Should I fulfil the ones that are missing?

Yes please! Don't worry so much about the ones using the `roblog` package (I can do those for you), but make sure to add links to the reviewers, that the package version listed in the YAML is up-to-date, and the tags (I can help with these if you're not sure). 

Then just ping me (@steffilazerte) when you're ready for review!

jospueyo commented:
Ready to review, @steffilazerte.

steffilazerte commented:
Great thanks, I'll take a look and get back to you soon!

steffilazerte reviewed (commented):
Hi @jospueyo 

This is lovely, short and sweet :) I have a couple of comments but the only main one is that it'd be nice to have a sentence or two concluding the article.

Thanks for contributing a blog post! I've included a draft tweet/toot for when we publish. Let me know if you have any suggestions

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [ ] title is in Title Case
* [ ] publication date is ok
* [X] alternative text of images is informative
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on [index.md](http://index.md/)
* [X] I ran `roblog::ro_check_urls()` on [index.md](http://index.md/)
* [X] I ran a spell-check on [index.md](http://index.md/)
* [ ] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary


---
Draft tweet/toot 

```
[technote] CRediTas is now part of rOpenSci 🎉

@jospueyo/@jospueyo@fosstodon.org shares a new R package 📦 for creating CRediT author statements. 

Simplify your academic writing!

https://ropensci.org/blog/2023/05/16/creditas-is-now-part/

#RStats
```

jospueyo commented on content/blog/2023-04-28-creditas-is-now-part-of-ropensci/index.Rmd:
```diff
@@ -0,0 +1,75 @@
+- Software Peer Review
+- CRediTas
+package_version: 0.2.0
+description: CRediTas is a package to generate CrediT author statements for scientific publications.
```
@steffilazerte I pushed the folder name change and now cannot commit suggestion. I integrate all of them in local commits.

jospueyo commented:
> Hi @jospueyo
> 
> This is lovely, short and sweet :) I have a couple of comments but the only main one is that it'd be nice to have a sentence or two concluding the article.
> 
> Thanks for contributing a blog post! I've included a draft tweet/toot for when we publish. Let me know if you have any suggestions
> 
> * [x]  post follows Content Guidelines
> * [x]  post follows Style Guide
> * [ ]  title is in Title Case
> * [ ]  publication date is ok
> * [x]  alternative text of images is informative
> * [x]  author metadata is provided with correct folder name
> * [x]  html not included in pull request of Rmd post
> * [x]  I ran `roblog::ro_lint_md()` on [index.md](http://index.md/)
> * [x]  I ran `roblog::ro_check_urls()` on [index.md](http://index.md/)
> * [x]  I ran a spell-check on [index.md](http://index.md/)
> * [ ]  YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
> * [x]  YAML field 'preface' is present if necessary
> 
> Draft tweet/toot
> 
> ```
> [technote] CRediTas is now part of rOpenSci 🎉
> 
> @jospueyo/@jospueyo@fosstodon.org shares a new R package 📦 for creating CRediT author statements. 
> 
> Simplify your academic writing!
> 
> https://ropensci.org/blog/2023/05/16/creditas-is-now-part/
> 
> #RStats
> ```

I already updated all your suggestions. Should I do something about the toot suggestion?

steffilazerte commented on content/blog/2023-05-16-creditas-is-now-part-of-ropensci/index.Rmd:
```diff
@@ -0,0 +1,78 @@
+
+More information on [the package vignette](https://docs.ropensci.org/CRediTas/articles/get_started.html).
+
+Hopefully, this simplifies your life as it simplified mine and allows you to spend more time on actually writing your articles. Also, to better manage your co-authors contributions. Please report any issues you have!
```
```suggestion
Hopefully, this simplifies your life as it simplified mine and allows you to spend more time on actually writing your articles. Also, to better manage your co-authors contributions. Please [report](https://github.com/ropensci/CRediTas/issues) any issues you have!
```

steffilazerte commented:
Hi @jospueyo, thanks for the speedy changes! 

If you're happy with the tweet/toot, then there is no further action needed from you on that matter (I just like to make sure I haven't misrepresented anything and sometimes authors have a better idea of how to present their work than I do).

I added one final suggestion to link to the GitHub issues in the last sentence, other than that we're good to go and I'll take it from here.

Thanks again!

jospueyo commented:
@steffilazerte 

> Hi @jospueyo, thanks for the speedy changes!
> 
> If you're happy with the tweet/toot, then there is no further action needed from you on that matter (I just like to make sure I haven't misrepresented anything and sometimes authors have a better idea of how to present their work than I do).
> 
> I added one final suggestion to link to the GitHub issues in the last sentence, other than that we're good to go and I'll take it from here.
> 
> Thanks again!

Ready to go! @steffilazerte

steffilazerte commented:
Great! Thanks again, it'll be published Tuesday morning Central North American time :grin: 
Take care!
